src/extra/AutoUI/utils.ts

```diff
@@ -71,3 +77,41 @@ export const autoUIGetDisabledReason = <T extends AutoUIBaseResource<T>>(
 		});
 	}
 };
+
+export const formatSorters: Dictionary<TableSortFunction<any>> = {};
+
+export const getSortingFunction = <T extends any>(
+	schemaKey: keyof T,
+	schemaValue: JSONSchema,
+): TableSortFunction<T> => {
+	if (formatSorters[schemaValue.format ?? '']) {
+		return formatSorters[schemaValue.format ?? ''];
+	}
```

Comment:
Like this is probably more readable 

```suggestion
	if (schemaValue.format && formatSorters[schemaValue.format]) {
		return formatSorters[schemaValue.format];
	}
```